### DISCUSSION TITLE
How to allow users to add group members?

### DISCUSSION THREAD

**Discussion by oculos** [keycloak/keycloak#29740] [Q&A]
Hi,

I have created a Realm role on the Master realm that has the following client roles for a given client:

manage-realm
manage-users
manage-authorization
query-users
query-groups
view-users

With that. the user can add a user to a group only by navigating to the user under "Users" menu. If the user goes to "Groups" and select a group, he/she cannot add the user there - the window doesn't allow him to add a user from there.

<img width="1150" alt="Screenshot 2024-05-21 at 13 33 56" src="https://github.com/keycloak/keycloak/assets/6791923/a2706098-d8d5-4248-9bfe-c5644a57e92c">

I noticed that when using Fine grained authz, this works, but here I want to give a global permission for the user to add users, so I am not using fine grained. 

Is there a way to accomplish this?

**Comment by col-panic** [comment]
If its generally possible to add the user to the group, but not in the way you mentioned, then I guess thats a bug. Maybe you should file an issue on this? Do you really need `manage-realm` and `manage-authorization` for this or can you narrow it down to the real `manage-*` you need?
